build.gradle

```diff
@@ -56,6 +59,10 @@ task wrapper(type: Wrapper) {
     gradleVersion = '4.4'
 }
 
+cobertura {
+    coverageFormats = ['html', 'xml']

```

Comment:
should travis gen html

Reply:
I left this in for local development.  I don't think it _really_ matters either way, but I find it interesting.